docs/cookbook/payments/customizing-credit-memo.rst

```diff
@@ -21,6 +21,17 @@ The first exemplary customization is to change background color of the heading o
             /* ... */
         </style>
 
+Customizing Credit Memo's logo
+----------------------------------
+
+Credit Memo's logo is by default displayed as Sylius logo.
+Change it by modifying the ``SYLIUS_REFUND_LOGO_FILE`` environment variable.
```

Comment:
```suggestion
Changing it is done by modifying the ``SYLIUS_REFUND_LOGO_FILE`` environment variable.
```

Reply:
why? :D much more formal tone than in our docs usually.